.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Introduction 
```

Comment:
The title ("Contribution to {admiral}") of the page got lost. Could we get it back?